fecgov/openFEC
# Feature/switch sched a

## Summary (required)

After the switch, the schedule_b queries will be pointed to fecp-driven fec_fitem_sched_a tables. When run the application locally, try api for schedule_a. 

- Addresses # 3088

## How to test the changes locally
After the switch, the schedule_b queries will be pointed to fecp-driven fec_fitem_sched_a tables. When run the application locally, try api for schedule_a
https://api.open.fec.gov/developers/#!/receipts/get_schedules_schedule_a
Make sure the API is still working.

## Impacted areas of the application
List general components of the application that this PR will affect:
This issue will impact schedule_a related API. However, the changes should be transparent to the users.
  



## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
